src/tasks/selfSell.ts

```diff
@@ -426,22 +459,48 @@ async function prepareWithdrawals({
     ));
   }
 
-  // TODO: add eth withdrawal
+  // TODO: remove once native asset orders are fully supported.
+  assertNotBuyingNativeAsset(toTokenAddress);
```

Comment:
This is supported now, we might want to try 😄

Reply:
There are a few edge cases with this: for example, if you are selling WETH for ETH this is going to trigger an error and should be handled in this script instead.

Reply:
Except for this, the withdraw script works! I briefly discussed the result of my test [here](https://cowservices.slack.com/archives/C0375NV72SC/p1676998332412359) and why we shouldn't actually do it.